### PR TITLE
Issue #117 Add EAP CD imagestreams and templates

### LINKED ISSUE #117: Add EAP CD imagestreams and templates
Add EAP CD templates and imagestreams from the following url
https://github.com/jboss-container-images/jboss-eap-7-openshift-image/tree/eap-cd/templates

See https://issues.jboss.org/browse/CDK-297

### PR DESCRIPTION
**Pull request by budhrg**
Fix #117

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
@budhrg LGTM, except a `readme`. But `readme` is up to you.

File: test/integration/features/addon-eap-cd.feature.bkp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    And with up to "10" retries with wait period of "500ms" the "status code" of HTTP request to "<http-endpoint>" of service "<service-name>" in namespace "myproject" is equal to "200"
+    And executing "oc delete all --all" succeeds
+
+    Examples: Required information to test the templates
+      | template-name           | service-name   | http-endpoint | expected-hello                        |
+      | eap-cd-basic-s2i        | eap-app        | /             | Welcome to JBoss!                     |
+
+  Scenario: User deletes Minishift
+    When executing "minishift delete --force" succeeds
+    Then Minishift should have state "Does Not Exist"
```
Comment (LalatenduMohanty) [inline test/integration/features/addon-eap-cd.feature.bkp]: @budhrg I guess this is a mistake to have `test/integration/features/addon-eap-cd.feature.bkp` in the PR.

File: test/integration/features/addon-eap-cd.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@addon-eap-cd @addon
+Feature: eap-cd add-on
+  Eap-cd add-on imports eap-cd imagestreams and templates, which are then available in OpenShift to the user.
+
+  Scenario: Installing the add-on
+    Given stdout of command "minishift addons list" does not contain "cors"
+     When executing "minishift addons install ../../add-ons/eap-cd" succeeds
```
Comment (agajdosi) [inline test/integration/features/addon-eap-cd.feature]: Smart!

Comment (budhrg) [reply test/integration/features/addon-eap-cd.feature]: :)

**Pull request merged**